app/javascript/pages/components/Calendar.jsx

```diff
@@ -111,25 +112,30 @@ class Calendar extends React.Component {
 
       function labels(svg) {
         let label = svg.append('g')
-          .style('font', 'bold 12px var(--sans-serif)')
+          .style('font', 'bold $font_size-base var(--sans-serif)')
```

Comment:
Check to see if this worked. I'm not sure if jsx will compile sass variables or not.

Reply:
changed to 12px